helloworld.py

```diff
@@ -0,0 +1,73 @@
+
+from flask import Flask, request
+import uuid
+import json
+import milvus_model
+from pymilvus import MilvusClient, DataType, Collection
+
+import boto3
+import os
+
+from dotenv import load_dotenv
+
+client = MilvusClient(
+    uri=os.getenv("MILVUS_ENDPOINT")
+)
+
+endpoint_url = "http://" + os.getenv("BUCKET_HOST") + ":"+ os.getenv("BUCKET_PORT")
+
+s3 = boto3.client(
+    "s3",
+    aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
+    aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
+    endpoint_url=endpoint_url,
+    use_ssl=False,
+    verify=False,
+)
+# Store bucket name
+bucket_name = os.getenv("BUCKET_NAME")
+
+collection_name = os.getenv("COLLECTION_NAME")
```

Comment:
should be the same as bucket name (maybe with some prefix to prevent name collisions)